build.gradle

```diff
@@ -132,6 +160,43 @@ configure(testProjects) {
   }
 }
 
+gradle.buildFinished {
+  String overallSummary = "Total Summary" +
+          "\n" +
+          "Results: ${(failedTestCount == 0) ? "SUCCESS": "FAILURE"} " +
+          "(" +
+          "${rootProject.testCount} tests, " +
+          "${rootProject.successfulTestCount} successes, " +
+          "${rootProject.failedTestCount} failures, " +
+          "${rootProject.skippedTestCount} skipped" +
+          ") " +
+          "in ${rootProject.time}" +
+          "\n"
+  rootProject.testsResults.add(overallSummary)
+
+  def allResults = rootProject.testsResults
+  if (!allResults.isEmpty()) { // allResults will be empty if we didn't run tests
+    // print results
+    printResults rootProject.testsResults
+  }
+}
+
+private static void printResults(allResults) {
+  println "\n  \n Build Summary:"
+  // Max line length has to take lines from all items into account
+  def maxLength = allResults*.readLines().flatten().collect { it.length() }.max()
+
+  println "┌${"${"─" * maxLength}"}┐" // Top border
```

Comment:
Use ASCII in source code. 
We use too many different tools and OSes to expect UTF-8 to work consistently.
Non-ASCII characters can be encoded using unicode escape sequences.

Reply:
Ok sure Joe, I'll search that up and try to find the solution!